deploy/template.yaml

```diff
@@ -278,18 +330,31 @@ Resources:
 
   SessionFunctionPermission:
     Type: AWS::Lambda::Permission
+    DependsOn:
+      - SessionFunction
     Properties:
       Action: lambda:InvokeFunction
       FunctionName: !GetAtt SessionFunction.Arn
       Principal: apigateway.amazonaws.com
 
   PostcodeLookupFunctionPermission:
     Type: AWS::Lambda::Permission
+    DependsOn:
```

Comment:
these permissions seem to be working on main without this DependsOn, so can we remove?